slack-pi-command/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "slack-pi-command",
+  "version": "1.0.0",
+  "private": true,
+  "license": "MIT",
+  "description": "Compute PI using a slack command",
+  "main": "function.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/binaris/functions-examples.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "bugs": {
+    "url": "https://github.com/binaris/functions-examples/issues"
+  },
+  "homepage": "https://github.com/binaris/functions-examples",
```

Comment:
nit: can use this example readme (https://github.com/binaris/functions-examples/slack-pi-command/README.md#slack-π-command-nodejs),

Reply:
Understood.